.github/workflows/tests.yml

```diff
@@ -103,3 +103,33 @@ jobs:
           # Warning: This is PowerShell syntax, not bash!
           g2p convert est fra fra-ipa > out
           if (diff (echo ɛ) (cat out)) { throw "Output did not match reference" }
+
+  test-heroku-env:
+    # Replicate what heroku will run
+    runs-on: ubuntu-22.04  # https://devcenter.heroku.com/articles/heroku-22-stack
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get the Python version from runtime.txt, like Heroku does
+        id: runtime
+        run: echo "PYTHON_VERSION=$(cat runtime.txt | sed 's/python-//')" >> $GITHUB_ENV
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - run: python -m pip install --upgrade pip
+      - name: Start with requirements.txt
```

Comment:
Pretty sure this is totally redundant with the other call to `pip install -r requirements.txt` below!

Reply:
Nearly: in testing, what I saw is that not having it there, pip would install stuff with `.[test]` and then have to uninstall some of it to install the right version. With this duplicated call up first, we avoir the version dance, but we redundantly check the version twice. I guess that warrants a comment in the code, since I agree it's obviously redundant in terms of the final outcome, and only done for optimization purposes.